### ISSUE TITLE
Add support for 2FA

### ISSUE THREAD

**Issue by jwbensley** [bgptools/issues#164]
URL:
```
https://bgp.tools/login
```

Browser (User-Agent if you can): 
```
Any / NA
```

Error base64 blob (If a site error):
```
N/A
```

What were you expecting:
```
To be able to enable 2FA under my user settings
```

What happened:
```
I am not able to configure 2FA / it's not supported
```

**N.B.** _This is a feature request not a bug, I'm not sure if there is a better way to request a feature so putting this here. Please direct me somewhere else if this is wrong._

I don't use PeeringDB SSO because I want to reduce the blast radius of the account if it is compromised. Instead, I use a dedicated account for logging in to https://bgp.tools however, I am unable to enable 2FA on https://bgp.tools.

Is it possible to get 2FA support added?

**Comment by benjojo** [comment]
I agree I would like to implement it, I do however need to fully plan out the account recovery story (aka, the hard bit of 2FA)
